charts/quickwit/values.yaml

```diff
@@ -98,17 +104,28 @@ indexer:
   livenessProbe:
     httpGet:
       path: /
-      port: http
+      port: rest
 
   readinessProbe:
     httpGet:
       path: /
-      port: http
+      port: rest
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+  persistentVolume:
+    enabled: false
+    # storage: "1Gi"
+    # storageClass: ""
 
 metastore:
   replicaCount: 1
 
-  # Extra env for metastore
+  # Extra env for searcher
```

Comment:
did you mean for `# Extra env for metastore`?

Reply:
yep, will fix that